plugins/woocommerce/client/admin/client/order-attribution-install-banner/use-order-attribution-install-banner.js

```diff
@@ -47,27 +97,82 @@ export const useOrderAttributionInstallBanner = () => {
 		[ currentUserCan ]
 	);
 
-	const { loading, isBannerDismissed } = useSelect( ( select ) => {
-		const { getOption, hasFinishedResolution } =
-			select( OPTIONS_STORE_NAME );
+	const { loading, isBannerDismissed, remoteVariantAssignment } = useSelect(
+		( select ) => {
+			const { getOption, hasFinishedResolution } =
+				select( OPTIONS_STORE_NAME );
 
-		return {
-			loading: ! hasFinishedResolution( 'getOption', [
-				OPTION_NAME_BANNER_DISMISSED,
-			] ),
-			isBannerDismissed: getOption( OPTION_NAME_BANNER_DISMISSED ),
-		};
-	}, [] );
+			return {
+				loading: ! hasFinishedResolution( 'getOption', [
+					OPTION_NAME_BANNER_DISMISSED,
+				] ),
+				isBannerDismissed: getOption( OPTION_NAME_BANNER_DISMISSED ),
+				remoteVariantAssignment: getOption(
+					OPTION_NAME_REMOTE_VARIANT_ASSIGNMENT
+				),
+			};
+		},
+		[]
+	);
+
+	const { loadingRecommendations, recommendations } = useSelect(
+		( select ) => {
+			const { getMiscRecommendations, hasFinishedResolution } =
+				select( STORE_KEY );
+
+			return {
+				loadingRecommendations: ! hasFinishedResolution(
+					'getMiscRecommendations'
+				),
+				recommendations: getMiscRecommendations(),
+			};
+		},
+		[]
+	);
+
+	const percentages = useMemo( () => {
+		if ( loadingRecommendations || recommendations.length === 0 ) {
```

Comment:
Maybe we can add `recommendations?.length` just to be sure recommendations has the length property.